packages/x-charts-pro/src/BarChartPro/BarChartPro.tsx

```diff
@@ -51,10 +51,10 @@ const BarChartPro = React.forwardRef(function BarChartPro(props: BarChartProProp
       <g {...clipPathGroupProps}>
         <BarChartPlotZoom {...barPlotProps} />
         <ChartsOverlay {...overlayProps} />
+        <ChartsAxisHighlight {...axisHighlightProps} />
       </g>
       <ChartsAxis {...chartsAxisProps} />
       <ChartsLegend {...legendProps} />
-      <ChartsAxisHighlight {...axisHighlightProps} />
       {!props.loading && <ChartsTooltip {...tooltipProps} />}
```

Comment:
Just to share some though: We immediatly thaught about moving this inside the `<g />` with clipping. But we can also have multiple groups referring to the same clipping path